src/components/no-content/no-content-component.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+
+import monete from 'assets/icon-pngs/monete.png';
+import styles from './no-content-styles.scss';
+
+const NoContent = ({ className, message, icon, minHeight }) => (
```

Comment:
duplicate `NoContent` component, I'm guessing that `no-content` is the adjusted one